src/sharedHelpers/parseExif.js

```diff
@@ -52,7 +52,7 @@ interface ExifToWrite {
 export const writeExifToFile = async ( photoUri: ?string, exif: ExifToWrite ): Promise<Object> => {
```

Comment:
Since this no longer writes arbitrary EXIF, it should be renamed `writeLocation` and accept only the arguments it is capable of writing, e.g. `latitude` and `longitude` (can we write horizontal accuracy to a PHAsset?).

Reply:
I think we should consider switching to [react-native-exify](https://github.com/lodev09/react-native-exify/) library - even though it's new, it does everything we need, including support for both `PHAsset` and regular URIs - for example here:
https://github.com/lodev09/react-native-exify/blob/main/ios/ExifyUtils.swift

What do you think?

Reply:
It doesn't seem to write `GPSHPositioningError` in iOS, nor does it read it in iOS when we can access it (i.e. when we're not going through PHAssets), so I'd be up for using it if you want to fork it and add that functionality. Otherwise I'd prefer to work on our own library. There's a high probability that if we start using `react-native-exify` we're going to have to end up supporting a fork of it when it gets abandoned anyway.

Reply:
Mmmm, true, didn't notice that issue - OK, I'll do the changes on our end